components/activity-section.jsx

```diff
@@ -0,0 +1,41 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+
+var ActivitySection = React.createClass({
+  propTypes: {
+    title: React.PropTypes.string.isRequired,
+    activities: React.PropTypes.array.isRequired
+  },
+  render: function () {
+    return (
+      <section className="activity-section">
+        <h2>
+          Section {this.props.title}
+        </h2>
+        {this.props.activities.map(function (activity, i) {
+          return (
+            <div className="activity" key={i}>
+              <Illustration height={122} width={122}
+              src1x={activity.image1x}
+              src2x={activity.image2x}
+              alt=""
```

Comment:
As I [mentioned elsewhere](https://github.com/mozilla/teach.webmaker.org/pull/980#issuecomment-108116906), this is actually problematic for a11y because it's a hyperlinked image... one solution may be to set `alt={activity.title}`. Redundant and annoying, but better than empty link text.